app/cryptography/jwt_encoder.py

```diff
@@ -27,22 +29,28 @@ def __init__(self):
     def encode(self, payload):
         return jwt.encode(payload, self.private_key, algorithm="RS256", headers={'kid': KID, 'typ': 'jwt'})
 
-    def encrypt_token(self, text, jwe_protected_header=None, encrypted_key=None, tag=None):
+    def encrypt_token(self, text, jwe_protected_header=None, cek=None, iv=None, encrypted_key=None, tag=None):
```

Comment:
Removing `cek` and `iv` should be done in another pull request, unless it's required by the pylint fixes?